run_single_test.sh

```diff
@@ -23,12 +23,19 @@ export VERBOSE_LOG=true
 export LOG_HTTP_REQUEST_SUMMARY=true
 export LOG_FAILED_HTTP_REQUESTS=true
 # logs output of raven server to stdout, helpful for failing tests
-# export LOG_RAVEN_SERVER=true
+#export LOG_RAVEN_SERVER=true
 #export PCAP_CAPTURE=true
 export LOG_ALL_REQUESTS=true
 #export ENABLE_FAILING_TESTS=true
 #export ENABLE_FLAKY_TESTS=true
 
+# must use full absolute path because working directory is direrectory of
+# ravendb server executable
+wd=`pwd`
+export RAVENDB_JAVA_TEST_CERTIFICATE_PATH="${wd}/certs/server.pfx"
```

Comment:
Can we generate these on a temp directory, instead?

Reply:
Then you can use them more easily, and we won't have to worry about leaking secrets